src/utils/Response.ts

```diff
@@ -1,8 +1,35 @@
+import { HttpCodesEnum } from "./HttpCodesEnum";
+
 export class Response {
```

Comment:
This should probably extend or in some way wrap `APIGatewayProxyResult`.  As it stands we wont know if our class drifts from the interface AWS provide.